### ISSUE TITLE
Relative file path changes depending on what file an experiment is called from

### ISSUE THREAD

**Issue by ScriBanana** [gama-platform/gama#3629]
Hi,

**Describe the bug**
Importing a GAML file with an experiment makes it so you can run it from the both its definition and importing file. (Example : If I define an experiment "Run" in `Experiment.gaml` and use `import "Experiment.gaml"` in another file, I will have the ![image](https://user-images.githubusercontent.com/72523821/220095850-169bd23c-57a3-465b-aca7-ffee0d4ce803.png) button over both in the UI.)
The issue is that any relative path called in the model will depend on the file the button is pressed from.

**To Reproduce**
Build a file system like this :
```
├── Models
│   ├── Experiments
│   │   └── Experiment.gaml
│   └── Main.gaml
└── Utilities
    └── GenerateExportFiles.gaml
```
with `main.gaml` :
```java
model Main
import "../Utilities/GenerateExportFiles.gaml"
import "Experiments/Experiments.gaml"
```
`Experiment.gaml` :
```java
model Experiments
import "../Main.gaml"
experiment run;
```
`GenerateExportFiles.gaml` :
```java
model GenerateExportFiles
global {
	string outputDirectory <- "../OutputFiles/";
	init {
		save "I am data!" to: outputDirectory + "Data.csv" type: csv rewrite: true;
	}
}
```

Here's the result :
```zsh
├── models
│   ├── Experiments
│   │   └── Experiments.gaml
│   ├── Main.gaml
│   └── OutputFiles
│       └── Data.csv
├── OutputFiles
│   └── Data.csv
└── Utilities
    └── GenerateExportFiles.gaml
```

**Expected behavior**
The `OutputFiles` directory being only created at the root of the project, at the location the path relative to where the `save` function was called from :
```zsh
├── models
│   ├── Experiments
│   │   └── Experiments.gaml
│   └── Main.gaml
├── OutputFiles
│   └── Data.csv
└── Utilities
    └── GenerateExportFiles.gaml
```

**Desktop (please complete the following information):**
- OS: Manjaro Linux x86_64
- GAMA version: 1.8.2 RC2

I could simply not cross-import gaml files as a workaround, but it still looks like an issue.

**Comment by AlexisDrogoul** [comment]
Actually, this organisation is more a feature than an issue, although I can understand why it might generate difficulties. It has been done so that support files and experiments could easily be defined together (so if you have relative paths in your model to access support files, the ones defined with the current experiment will be used).

**Comment by ScriBanana** [comment]
Hi Alexis,
Alright, I see that being useful in that use case. It makes sense when explained. Maybe it deserves an explanation in the documentation to avoid confusion like mine ?

**Comment by AlexisDrogoul** [comment]
You are right. It definitely deserves a more detailed explanation somewhere... Let us ask @RoiArthurB where it could go in the documentation 🤔 ...
